runtime/gc_glue_java/MarkingDelegate.hpp

```diff
@@ -66,6 +68,7 @@ class MM_MarkingDelegate
 protected:
 
 public:
+	MM_StackSlotIteratorCallBackFunc _callbackFunction;
```

Comment:
I believe this is not needed. Just make  stackSlotIteratorForMarkingDelegate non-static and declare it above instead of introducing MM_StackSlotIteratorCallBackFunc type. Later just use it directly instead of _callbackFunction.